### PR TITLE
[hotfix] Fix regression / crash caused by #7873

### PR DESCRIPTION
**Pull request by erjiaqing**
#### Problem
there are some regressions caused by #7873
- Sending command to unexist endpoint causes crash
- Python device controller cannot receive response due to mismatch type
- Fixes ##7983

#### Change overview
- Fix type in src/controller/python/chip/interaction_model/delegate.py
- Add static assert to guard this.
- Fix `findClusterEndpointIndex`

#### Testing
- Manaully tested with `src/controller/python/test/test_scripts/mobile-device-test.py` (network commissioning not included for #7913

### CODE REVIEW

**Review by saurabhst — APPROVED** [review]

**Review by msandstedt — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

File: src/app/util/attribute-storage.cpp [hunk 1]
```diff
@@ -884,7 +884,7 @@ static uint16_t findClusterEndpointIndex(EndpointId endpoint, ClusterId clusterI
 
     if (emberAfFindClusterWithMfgCode(endpoint, clusterId, mask, manufacturerCode) == NULL)
     {
-        return 0xFF;
+        return 0xFFFF;
```
Comment (bzbarsky-apple) [inline src/app/util/attribute-storage.cpp:887]: We really should have added symbolic constants for all this stuff.  :(  @vivien-apple followup?

File: src/controller/python/chip/interaction_model/Delegate.h [hunk 2]
```diff
@@ -35,6 +37,11 @@ struct __attribute__((packed)) CommandStatus
     uint8_t CommandIndex;
 };
 
+static_assert(std::is_same<chip::EndpointId, uint16_t>::value && std::is_same<chip::ClusterId, uint32_t>::value &&
+                  std::is_same<chip::CommandId, uint32_t>::value,
+              "Members in CommandStatus does not match interaction_model/delegate.py");
+static_assert(sizeof(CommandStatus) == 4 + 2 + 2 + 4 + 4 + 1, "Size of CommandStatus might contains padding");
```
Comment (bzbarsky-apple) [inline src/controller/python/chip/interaction_model/Delegate.h:43]:
```suggestion
static_assert(sizeof(CommandStatus) == 4 + 2 + 2 + 4 + 4 + 1, "Size of CommandStatus might contain padding");
```

Comment (bzbarsky-apple) [inline src/controller/python/chip/interaction_model/Delegate.h:43]:
```suggestion
static_assert(sizeof(CommandStatus) == 4 + 2 + 2 + 4 + 4 + 1, "Size of CommandStatus might contain padding");
```

**Review by bzbarsky-apple — APPROVED** [review]
Can we please add a test for the "sending command to non-existent endpoint" case?  This should be eminently testable via the yaml stuff we have, if the crash is on the server....  Followup PR is fine for this.

**Review by mspang — APPROVED** [review]

File: src/app/util/attribute-storage.cpp (hunk 1 above)
Comment (vivien-apple) [reply src/app/util/attribute-storage.cpp:887]: @bzbarsky-apple That’s in my todo list!

**Pull request merged**